open-sphere-base/core/src/main/java/io/opensphere/core/map/MapManagerMenuProvider.java

```diff
@@ -46,6 +57,18 @@
     @SuppressWarnings("unused")
     private final boolean myShowToast = true;
 
+    /**
+     * The location to be displayed by the popup manager. Package visibility to
+     * prevent synthetic accessors.
+     */
+    LatLonAlt myLocation;
```

Comment:
If you understand why you're keeping package visibility for these two variables, that's fine. Otherwise, change it to whatever visibility is appropriate.

Reply:
Yea it needs to be package to ensure its the correct instance